### ISSUE TITLE
Could not load the configuration

### ISSUE THREAD

**Issue by KiranOtter** [Rem0o/FanControl.Releases#129]
Hello.  Been using FanControl for a while now and just today I discovered it wasn't working.  When I attempt to run it it gives the message that it 'could not load the configuration'.  If I start a new configuration and save it, it seems OK, but I'd rather not have to recreate the configuration I had. I looked at the .json file but I can't tell what's wrong with it. I'll attach the configuration file here in hopes it helps determine the issue.

Thanks!
[userConfig.zip](https://github.com/Rem0o/FanControl.Releases/files/5845911/userConfig.zip)

Update; I misspoke. It won't load any configuration, even new ones.  I'm on V63.

**Comment by KiranOtter** [comment]
I decided to start from scratch, download V63 anew, run it, save the config.  When I exit and try to open it again, it says it can't load the configuration.  I then tried it on another PC, and it works, so I'm not sure what's going on.. but it's obviously isolated to just my PC. I'll continue to try and troubleshoot.

Kiran

**Comment by Richie78321** [comment]
I also had an issue loading my existing config with v63. I rolled back to v62 and everything is working again.

**Comment by KiranOtter** [comment]
1/21/2021 12:01:23 PM: System.IO.FileNotFoundException: The system cannot find the file specified. (Exception from HRESULT: 0x80070002)
   at Microsoft.Win32.TaskScheduler.V2Interop.IRegisteredTask.get_Enabled()
   at FanControl.Domain.ApplicationContext.GetRegisterInStartup()
   at FanControl.ViewModels.MainWindowViewModel.a(SerializableMainWindow A_0, IFanCurveProvider`1 A_1)
   at FanControl.ViewModels.MainWindowViewModel.c()

**Comment by KiranOtter** [comment]
> 
> 
> I also had an issue loading my existing config with v63. I rolled back to v62 and everything is working again.

Thank you for the reply. I just took V59 from a backup and tried to run it and still have the same error.  I don't know what file it can't find.

**Comment by Rem0o** [comment]
> > I also had an issue loading my existing config with v63. I rolled back to v62 and everything is working again.
> 
> Thank you for the reply. I just took V59 from a backup and tried to run it and still have the same error. I don't know what file it can't find.

Seems like it is trying to read if the Task for Startup with Windows is enabled or not, and it fails at that point in the Task Scheduler itself. Try and delete the "FanControl" task in your Task Scheduler and report back.

**Comment by KiranOtter** [comment]
> Seems like it is trying to read if the Task for Startup with Windows is enabled or not, and it fails at that point in the Task Scheduler itself. Try and delete the "FanControl" task in your Task Scheduler and report back.

When I open the Task Scheduler, I get a message that the 'selected task 'FanControl" no longer exists. To see the current tasks, click Refresh.  I can click OK, and when I click refresh, I get the same message again.  So I checked the registry and found something in: Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Schedule\TaskCache\Tree\FanControl  I deleted that, and now the Task Scheduler stops saying the task no longer exists.  And now when I run FanControl, it works!

Thank you for pointing me in the right direction.  Not sure what happened with the scheduled task.

Kiran
